api/obs/api/routes/exports.py

```diff
@@ -119,3 +120,74 @@ async def export_events(req):
             return json_response(geojson)
 
         raise InvalidUsage("unknown export format")
+
+
+@api.get(r"/export/road_usage")
+async def export_road_usage(req):
+    async with use_request_semaphore(req, "export_semaphore", timeout=30):
+        bbox = req.ctx.get_single_arg(
+            "bbox", default="-180,-90,180,90"
+        )
+        assert re.match(r"(-?\d+\.?\d+,?){4}", bbox)
+        fmt = req.ctx.get_single_arg("fmt", convert=ExportFormat)
+        events = await req.ctx.db.stream(
+            text(
+                f"select ST_AsGeoJSON(ST_Transform(geometry,4326)), way_id, distance_overtaker_mean, distance_overtaker_min,distance_overtaker_max,distance_overtaker_median,overtaking_event_count,usage_count,direction,zone,offset_direction,distance_overtaker_array from layer_obs_roads(ST_Transform(ST_MakeEnvelope({bbox},4326),3857),11,NULL,'1900-01-01'::timestamp,'2100-01-01'::timestamp) WHERE usage_count>0"
```

Comment:
Use `AS transformed_geometry` or so to avoid the strange and possibly undefined-behaviour access via `road.st_asgeojson` below.

Reply:
done